index.js

```diff
@@ -235,33 +234,35 @@ function schemaMiddleware (schema) {
 }
 
 function cors (opts) {
-  var _cors = corsify(opts)
+  opts = opts || {}
+  var headers = opts.headers || ['Content-Type', 'Accept', 'X-Requested-With']
+  var methods = opts.methods || ['PUT', 'POST', 'DELETE', 'GET', 'OPTIONS']
```

Comment:
⚠️ this does not do what you think it does.

From [the node docs](https://nodejs.org/api/http.html#http_response_setheader_name_value):
> Sets a single header value for implicit headers. If this header already exists in the to-be-sent headers, its value will be replaced. Use an array of strings here if you need to send multiple headers with the same name.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS):
```txt
Access-Control-Allow-Origin: http://foo.example
Access-Control-Allow-Methods: POST, GET, OPTIONS
Access-Control-Allow-Headers: X-PINGOTHER, Content-Type
Access-Control-Max-Age: 86400
```

The array needs to be concatenated and comma separated. We also need to explicitly handle the case of a single value too.